### DISCUSSION TITLE
Unable to read the tetrahedalization of the scene mesh file, calculation canceled.

### DISCUSSION THREAD

**Discussion by Savilix** [Ifsttar/I-Simpa#362] [Q&A]
Hello everyone,
I imported a 3ds file in i simpa made with Rhino 7 and I always get this error "Unable to read the tetrahedalization of the scene mesh file, calculation canceled." How do I solve?

I state that I have redone the model many times but to no avail....

**Comment by nicolas-f** [comment]
Hi.

This is caused by overlaps/intersecting faces in your model.

Please provide the 3D model file in **stl** or **ply** in order to be able to help you.

**Reply by lilismybao** [reply]
Hello, I’m facing the same issue, with the difference that I exported it from SketchUp. I would like to know if there is a step-by-step guide to solve this, if I can fix it directly in i-Simpa, or if it would be better to remodel everything from scratch in SketchUp.

**Reply by nicolas-f** [reply]
Hi,

You can repair your model in FreeCad:
https://www.youtube.com/watch?v=3fcS7E_UyLg

You can try to mesh it to check if it works directly in freecad then export the mesh to I-Simpa.
